src/commonTest/kotlin/com/ucasoft/kcron/builders/BuilderTests.kt

```diff
@@ -10,6 +10,7 @@ import io.kotest.matchers.shouldBe
 import kotlinx.datetime.*
 import kotlin.test.Test
 
+@OptIn(DelicateIterableApi::class)
 class BuilderTests {
 
     @Test
```

Comment:
Also add **new** test where expression will start from `2050`, for example, but `asIterable()` will be called from `2060`.